proposals/NNNN-android_o_changes_phase_1.md

```diff
@@ -0,0 +1,76 @@
+# Android O Changes Phase 1
+
+* Proposal: [SDL-NNNN](NNNN-android_o_changes_phase_1.md)
+* Author: [Joey Grover](https://github.com/joeygrover)
+* Status: **Awaiting review**
+* Impacted Platforms: [Android]
+
+## Introduction
+
+Android O introduces a lot of new restrictions in terms of background services and when an app becomes activated. This proposal intends to provide enhancements that will work within the new Android O environment, specifically implicit intents and static broadcast receivers.
+
+## Motivation
+The way the sdl_andorid library currently works will not be functional if the developer targets their app for Android O. There are a few changes we need to make to ensure compatibility, this is the first phase of those changes. The change introduce in the newest version of Android that this proposal is aiming to handle is Android's reduction in accepted implicit intents for static broadcast receivers. An implicit intent is defined by the intent merely having an "action" field, with no specific destination. Previous versions of Android would allow developers to define static broadcast receivers in their manifests that could listen for an intent with that specific "action". The  only implicit intents that developers can use a static broadcast receiver with will be from the Android OS and defined [here](https://developer.android.com/preview/features/background-broadcasts.html). (Note: dynamic broadcast receivers can still receive implicit intents, however, they are subject to the lifecycle of the component in which they are created.)
+
+The current SDL Android library uses an implicit intent to broadcast the `SdlRouterService` is currently connected to a module. It is assumed the developers listen for this intent in a static broadcast receiver so the app can be woken up. 
```

Comment:
Can you please clarify this first sentence?  Should it read "...the `SdlRouterService`currently..." (removing "is")?